bin/test_exercises.sh

```diff
@@ -93,8 +93,14 @@ do
       module_name=$(cat "${test_file}" | sed -rn 's/^defmodule (.*)Test do$/\1 /p')
       doctest_code="doctest ${module_name}"
 
-      # .bak and rm part left here to simplify GNU/BSD sed options compatibility
-      sed -i.bak 's/use ExUnit.Case\(.*\)/use ExUnit.Case\1\n'" ${doctest_code}"'\n/g' "${test_file}" && rm -f "${test_file}.bak"
+      if [ ${module_name} != "Form" ]
```

Comment:
I'm not super comfortable with hardcoding a file name. 
Maybe detecting the use of doctests in the each file before using `sed` would be better?

Reply:
I thought about this a bit more...
The problem is that there would be two `doctest ModuleName` in a row right? Maybe we can simply use `sed` again to remove double appearances? Is that too clunky?